src/pages/tasks/NewTaskPage.tsx

```diff
@@ -181,33 +163,33 @@ function NewTaskPage(props) {
-                                label={assignee.displayName ? props.translate('task.assignee') : ''}
-                                title={assignee.displayName || ''}
-                                description={assignee.displayName ? LocalePhoneNumber.formatPhoneNumber(assignee.subtitle) : props.translate('task.assignee')}
-                                icon={assignee.icons}
+                                label={assignee?.displayName ? translate('task.assignee') : ''}
+                                title={assignee?.displayName ?? ''}
+                                description={assignee?.displayName ? LocalePhoneNumber.formatPhoneNumber(assignee?.subtitle) : translate('task.assignee')}
+                                icon={assignee?.icons}
                                 onPress={() => Navigation.navigate(ROUTES.NEW_TASK_ASSIGNEE)}
                                 shouldShowRightIcon
                                 titleWithTooltips={assigneeTooltipDetails}
                             />
                             <MenuItem
-                                label={shareDestination.displayName ? props.translate('newTaskPage.shareSomewhere') : ''}
-                                title={shareDestination.displayName || ''}
-                                description={shareDestination.displayName ? shareDestination.subtitle : props.translate('newTaskPage.shareSomewhere')}
-                                icon={shareDestination.icons}
+                                label={shareDestination?.displayName ? translate('newTaskPage.shareSomewhere') : ''}
+                                title={shareDestination?.displayName ?? ''}
+                                description={shareDestination?.displayName ? shareDestination.subtitle : translate('newTaskPage.shareSomewhere')}
+                                icon={shareDestination?.icons}
                                 onPress={() => Navigation.navigate(ROUTES.NEW_TASK_SHARE_DESTINATION)}
-                                interactive={!props.task.parentReportID}
-                                shouldShowRightIcon={!props.task.parentReportID}
-                                titleWithTooltips={!shareDestination.shouldUseFullTitleToDisplay && shareDestination.displayNamesWithTooltips}
+                                interactive={!task?.parentReportID}
+                                shouldShowRightIcon={!task?.parentReportID}
+                                titleWithTooltips={shareDestination?.shouldUseFullTitleToDisplay ? shareDestination?.displayNamesWithTooltips : []}
                             />
                         </View>
                     </View>
-                    <View style={[styles.flexShrink0]}>
+                    <View style={styles.flexShrink0}>
                         <FormAlertWithSubmitButton
-                            isAlertVisible={!_.isEmpty(errorMessage)}
+                            isAlertVisible={!isEmptyObject(errorMessage)}
```

Comment:
The way `isEmptyObject` is implemented with obj, it looks like `isEmptyObject` is `true` for any string. I think this is not what we expect here.

Reply:
Update to negation, since `isEmptyObject` is mostly for objects

But cannot agree. `isEmptyObject` will work well with empty and nonempty strings. (you can check it by yourself, `Object.keys()` works great with string and converting it into an empty or nonempty array )